src/types/index.ts

```diff
@@ -155,7 +155,7 @@ export interface StorageMappings {
 export type TenantPaymentProviders = 'external' | 'stripe';
 
 export interface Tenants {
-    collections_quota: number;
+    // collections_quota: number;
```

Comment:
Any reason not to remove this now?

Reply:
Since it is in the "root type` I thought it would be best to just comment out so people know it is there.